conanfile_installer.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+import os
+from conanfile_base import ConanfileBase
+
+
+class ConanfileInstaller(ConanfileBase):
+    name = "flex_installer"
+    version = ConanfileBase.version
+    exports = ConanfileBase.exports + ["conanfile_base.py"]
```

Comment:
it can be removed if https://github.com/bincrafters/conan-flex/pull/7/files#r279763014 is accepted

Reply:
forget about, it doesn't make sense if your base recipe is also your library recipe.